packages/wrangler/src/deploy/deploy.ts

```diff
@@ -26,6 +26,7 @@ import { addHyphens } from "../deployments";
 import { confirm } from "../dialogs";
 import { getMigrationsToUpload } from "../durable";
 import { UserError } from "../errors";
+import { syncAssets } from "../experimental-assets";
```

Comment:
Should we rename this `syncExperimentalAssets`?

It would conform to the `legacyAssets` / `experimentalAssets` naming convention and reduce confusion between the two.

When we stabilise this stuff, we can rename all experimental assets occurrences to just assets

Reply:
+1